pkg/secret/secret.go

```diff
@@ -54,7 +54,9 @@ func (s *secretParser) GetRepoSecret(path string) (map[string]string, error) {
 
 // GetOauthSecret parses the oauth secret
 func (s *secretParser) GetOauthSecret(path string) (*core.Oauth, error) {
-	o := &core.Oauth{}
+	o := &core.Oauth{
+		Type: core.Bearer,
```

Comment:
@utkarsh-lambdatest Please assess. I think this might break `Basic` auth in synapse bitbucket

Reply:
Json unmarshal will overwrite the type, right?